### DISCUSSION TITLE
Lora send but can't receive well

### DISCUSSION THREAD

**Discussion by hoseinaghajari** [sandeepmistry/arduino-LoRa#470] [Q&A]
Hi friends, **I wrote codes for simple a gateway and a node. gateway send a message to node. but after 5 or 6 times send and receive, the node is hanging and not receive from gateway.**
I attach the codes below:
**_sender:_**
![1](https://user-images.githubusercontent.com/37798557/119216564-322c9300-ba89-11eb-8022-f7e50f3f115d.JPG)

**_receiver:_**
![2](https://user-images.githubusercontent.com/37798557/119216573-3ce72800-ba89-11eb-8e2f-65b453a253f9.JPG)

**result is not permanent. gateway send 4 or 5 time to node but after that node can't receive anything.!!! I tried many changes in my code but at first lets discuss about this simple one.** please help me. thanks.

**Comment by IoTThinks** [comment]
Read the sample examples again.
Check sender and receiveCallback examples

**Comment by IoTThinks** [comment]
If already use the examples and still can not, try to provide external powers to the LoRa.
